README.md

```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_ios/releases/tag/7.3.2)
 
 ---
 
-#### SDL Core: 8.0.0
-
-**Release Date**: October 27, 2021
+#### SDL Java Suite: 5.4.0
+**Release Date**: April 13, 2022
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_core/releases/tag/8.0.0).
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_java_suite/releases/tag/5.4.0)
 
 ---
 
-#### SDL iOS: 7.3.0
-**Release Date**: October 27, 2021
+#### SDL JavaScript Suite: 1.6.0
+**Release Date**: April 13, 2022
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_ios/releases/tag/7.3.0)
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_javascript_suite/releases/tag/1.6.0).
 
 ---
 
-#### SDL Java Suite: 5.3.0
-**Release Date**: October 27, 2021
+#### SDL Policy Server: 3.0.0
+**Release Date**: April 13, 2022
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_java_suite/releases/tag/5.3.0)
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/3.0.0).
 
----
+--
 
-#### SDL JavaScript Suite: 1.5.0
-**Release Date**: October 27, 2021
+#### Manticore: 2.9.0
+**Release Date**: November 2021
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_javascript_suite/releases/tag/1.5.0).
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/manticore/projects/4). 
 
----
+--
 
-#### SDL Policy Server: 2.12.0
+#### SDL Protocol Spec: 5.4.1
 **Release Date**: October 27, 2021
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/2.12.0).
-
----
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/protocol_spec/releases/tag/5.4.1).
 
-#### Manticore: 2.9.0
-**Release Date**: December 8, 2021
+--
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/manticore/releases/tag/2.9.0). 
+#### SDL RPC Spec: 8.0.0
+**Release Date**: October 27, 2021
 
----
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/rpc_spec/releases/tag/8.0.0).
 
 ## Upcoming Releases
 
+#### SDL Core: 8.2.0
 
-#### SDL iOS: 7.4.0
-**Target Release Date**: April 13, 2022
+**Target Release Date**: October 26, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/27).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/22).
 
 ---
 
-#### SDL Java Suite: 5.4.0
-**Target Release Date**: April 13, 2022
+#### SDL iOS: 7.3.3
```

Comment:
```suggestion
#### SDL iOS: 7.4.0
```
We should plan for this to be a minor release as usual, and will update accordingly if we determine only a hotfix is needed.